lascli/parser/logs.py

```diff
@@ -1,8 +1,21 @@
+from datetime import datetime
 from las import Client
 
 
-def get_log(las_client: Client, log_id):
-    return las_client.get_log(log_id)
+def parse_log(response):
+    log_events = []
+    for event in response['events']:
+        timestamp = int(event['timestamp'])/1000
```

Comment:
This will become seconds on next update, I suggest removing the division and waiting for API update before publish